assets/src/pages/room/components/StreamPlayer/MediaPlayerPeersSection.tsx

```diff
@@ -0,0 +1,230 @@
+import React, { FC } from "react";
+import { RemotePeer, ApiTrack } from "../../hooks/usePeerState";
+import MediaPlayerTile from "./MediaPlayerTile";
+import { MembraneWebRTC, TrackEncoding } from "@membraneframework/membrane-webrtc-js";
+import clsx from "clsx";
+import { StreamSource, TrackType } from "../../../types";
+import InfoLayer from "./PeerInfoLayer";
+import PeerInfoLayer from "./PeerInfoLayer";
+
+export type TrackWithId = {
+  stream?: MediaStream;
+  trackId?: string;
+  encodingQuality?: TrackEncoding;
+  metadata?: any;
+  enabled?: boolean;
+};
+
+export type MediaPlayerTileConfig = {
+  peerId?: string;
+  emoji?: string;
+  flipHorizontally?: boolean;
+  displayName?: string;
+  video: TrackWithId[];
+  audio: TrackWithId[];
+  playAudio: boolean;
+  screenSharing: TrackWithId[];
+  showSimulcast?: boolean;
+  remoteSimulcast?: boolean;
+  streamSource: StreamSource;
+};
+
+const getTracks = (tracks: ApiTrack[], type: TrackType): TrackWithId[] =>
+  tracks
+    .filter((track) => track?.metadata?.type === type)
+    .map(
+      (track): TrackWithId => ({
+        stream: track.mediaStream,
+        trackId: track.trackId,
+        encodingQuality: track.encoding,
+        metadata: track.metadata,
+        enabled: true,
+      })
+    );
+
+const getCameraStreams = (peers: RemotePeer[], showSimulcast?: boolean): MediaPlayerTileConfig[] => {
```

Comment:
This function name confused me a bit as it gets more than just camera streams, can you please rename it?